pom.xml

```diff
@@ -121,14 +121,13 @@
                     <productVersion>${confluence.version}</productVersion>
                     <productDataVersion>${confluence.data.version}</productDataVersion>
                     <enableQuickReload>false</enableQuickReload>
+                    <extractDependencies>false</extractDependencies>
                     <instructions>
                         <Atlassian-Plugin-Key>${atlassian.plugin.key}</Atlassian-Plugin-Key>
                         <!-- Add package to export here -->
                         <Export-Package>com.baloise.confluence.digitalsignature.api</Export-Package>
                         <!-- Add package import here -->
                         <Import-Package>
-                            org.springframework.osgi.*;resolution:="optional",
-                            org.eclipse.gemini.blueprint.*;resolution:="optional",
```

Comment:
not sure what will be the impact, did you test?

Reply:
I've reverted both:
- `org.springframework.osgi.*` is required - otherwise enabling plugin features fails
- `org.eclipse.gemini.blueprint.*` might not be necessary, but it is part of the stub created with the atlassian plugin SDK

I've tested the installation and some basic features using Confluence 7.5.